staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -577,7 +577,16 @@ func (s *SecureServingInfo) HostPort() (string, int, error) {
 // AuthorizeClientBearerToken wraps the authenticator and authorizer in loopback authentication logic
 // if the loopback client config is specified AND it has a bearer token.
```

Comment:
update godoc to indicate that authn and authz must be non-nil and contain existing Authenticator and Authorizers, or the function no-ops

Reply:
> update godoc to indicate that authn and authz must be non-nil and contain existing Authenticator and Authorizers, or the function no-ops

sorry I was unclear... I didn't mean to remove the nil checks and early returns, just to document that if authn/authz are nil, the function doesn't add in a token authenticator or authorizer

Reply:
sry for misreading.. last change reverted and godoc updated. ptal @liggitt